lib/view/view.py

```diff
@@ -299,19 +299,32 @@ def info_XDR(stats, xdr_enable, cluster, timestamp="", **ignore):
 
     @staticmethod
     @reserved_modifiers
-    def info_sindex(stats, cluster, timestamp="", with_=None, **ignore):
+    def info_sindex(
+        sindex_stats, ns_configs, cluster, timestamp="", with_=None, **ignore
+    ):
         # return if sindex stats are empty.
-        if not stats:
+        if not sindex_stats:
             return
 
+        # Insert sindex-type from ns_confgs into sindex_stats. Sheets can't handle
+        # two objects with differing schemas.
+        for node, node_stats in sindex_stats.items():
+            for key in node_stats:
+                ns = key.split()[0]
+
+                sindex_type = ns_configs.get(node, {}).get(ns).get("sindex-type", None)
```

Comment:
No alternative value for .get(ns)?

Reply:
It's a tradeoff between future proofing and backwards compatibility